SAMtags.tex

```diff
@@ -57,6 +57,7 @@ \section{Standard tags}
   {\tt AS} & i & Alignment score generated by aligner \\
   {\tt BC} & Z & Barcode sequence \\
   {\tt BQ} & Z & Offset to base alignment quality (BAQ) \\
+  {\tt BX} & Z & Corrected sequence bases for the unique molecular identifier \\
```

Comment:
Could this reference the RX tag, i.e. "for the unique molecular identifier (`RX`).